kvmagent/kvmagent/key_agent/key_agent.proto

```
@@ -0,0 +1,44 @@
+syntax = "proto3";
+package keyagent.v1;
```

Buf (1.65.0)

[error] Files with package "keyagent.v1" must be within a directory "keyagent/v1" relative to root but were in directory "kvmagent/kvmagent/key_agent".

(PACKAGE_DIRECTORY_MATCH)